src/plugins/content_management/public/registry/content_type_definition.ts

```diff
@@ -39,4 +40,9 @@ export interface ContentTypeDefinition {
    * If not provided the default CRUD client is used assuming that this type has a server-side content registry
    */
   crud?: CrudClient;
+
+  version: {
+    /** The latest version for this content */
+    latest: Version;
```

Comment:
Would it be more current to call it "current" on the client? 
Because the client version would be "current" for the current client version, but the server could be on the later "latest" version?

Reply:
There are always both on the "latest" ... as far as they are concerned. 😊 So I would stick with consistency for CM providers on how they declare versions in both place.